biodata-models/src/test/java/org/opencb/biodata/models/variant/VariantVcfFactoryTest.java

```diff
@@ -531,5 +531,32 @@ public void testParseInfo() {
         assertEquals(10685, Integer.parseInt(getFile1.getAttribute("MQ")));
         assertEquals(1, Integer.parseInt(getFile1.getAttribute("MQ0")));
     }
-    
+
+    @Test
+    public void testCreateVariantWithEmptyIds() {
```

Comment:
This test should be renamed or split into 2 or 3, because it not only tests the case where a variant has an empty ID.